docs/operations/clone_api.md

```diff
@@ -0,0 +1,157 @@
+please see [Snapshot & Restore prerequesites](./snapshot_restore_api.md/#prerequesites).
+
+### Snapshot Feature Gate
+
+Currently, clone API is gaurded by Snapshot feature gate. The
+[feature gates](./activating_feature_gates.md#how-to-activate-a-feature-gate)
+field in the KubeVirt CR must be expanded by adding the `Snapshot` to it.
+
+## The clone object
+
+Firstly, as written above, the clone API relies upon Snapshot & Restore APIs under the hood. Therefore, it might be helpful
+to look at [Snapshot & Restore](./snapshot_restore_api.md) user-guide page for more info.
+
+### VirtualMachineClone object overview
+
+In order to initiate clone, a `VirtualMachineClone` object (CRD) needs to be created on the cluster. An example
+for such an object is:
+```yaml
+kind: VirtualMachineClone
+apiVersion: "clone.kubevirt.io/v1alpha1"
+metadata:
+  name: testclone
+
+spec:
+  # source & target definitions
+  source:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: vm-cirros
+  target:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: vm-clone-target
+
+  # labels & annotations definitions
+  labelFilters:
+    - "*"
+    - "!someKey/*"
+  annotationFilters:
+    - "anotherKey/*"
+
+  # other identity stripping specs:
+  newMacAddresses:
+    interfaceName: "00-11-22"
+  newSMBiosSerial: "new-serial"
+```
+
+In the next section I will go through the different settings to elaborate them.
+
+#### Source & Target
+
+The source and target indicate the source/target API group, kind and name. A few important notes:
+* Currently, the only supported kind is `VirtualMachine`, but more types are expected to be supported in the future.
+  See "future roadmap" below for more info.
+* The target name is **optional**. If unspecified, the clone controller will generate a name for the target automatically.
+
+#### Label & Annotation filters
+
+These spec fields are intended to determine which labels / annotations are being copied to the target or stripped away.
+
+The filters are a list of strings. Each string represents a key that may exist at the source. Every source key that matches
+to one of these values is being copied to the cloned target. In addition, special regular-expression-like characters can be
+used:
+
+* **Wildcard character (\*)** can be used to match to anything. Wildcard can be only used at the **end** of the filter.
+  * These filters are valid:
+    * "*"
+    * "some/key*"
+  * These filters are invalid:
+    * "some/*/key"
+    * "*/key"
+* **Negation character (!)** can be used to avoid matching certain keys. Negation can be only used at the **beginning** of a filter.
+  Note that a Negation and Wildcard can be used together.
+  * These filters are valid:
+    * "!some/key"
+    * "!some/*"
+  * These filters are invalid:
+    * "key!"
+    * "some/!key"
+
+Setting label / annotation filters is **optional**. If unset, all labels / annotations will be copied as a default.
+
+#### newMacAddresses
```

Comment:
Can we add a seciton called `Handling of identity` or some thing similar.

WHat happens during cloning to
- mac (covered below)
- smbios (covered below)
- disk serial numbers
- any volume (dv, pvc, secret, configmap, …)

Reply:
Ehum..
It's already covered under the description of the spec fields. I'm feeling like it would simply duplicate the information.

Regarding disk serial / volume cloning - this is being done with snapshot / restore which ultimately uses VolumeSnapshot/VolumeRestore, their implementation is different depending on your CSI driver.